pkg/virt-controller/watch/clone/clone.go

```diff
@@ -342,6 +352,11 @@ func (ctrl *VMCloneController) updateStatus(origClone *clonev1alpha1.VirtualMach
 		)
 	}
 
+	if syncInfo.pvcBound {
+		vmClone.Status.SnapshotName = nil
+		vmClone.Status.RestoreName = nil
```

Comment:
Can you help me understand why we do this?

Reply:
I think that the reason behind this was that the resources we are referring here does not exist anymore.
Currently, I don't have a strong opinion here, but it looks reasonable to me. WDYT?

Reply:
I have checked the code and I don't think this does anything. For me, it would be better to keep it.

Reply:
Actually, this is used here:
```
case clonev1alpha1.Succeeded:

		if vmClone.Status.RestoreName != nil {
			syncInfo = ctrl.verifyPVCBound(vmClone, syncInfo)
			if syncInfo.toReenqueue() {
				return syncInfo
			}

			syncInfo = ctrl.cleanupRestore(vmClone, syncInfo)
			if syncInfo.toReenqueue() {
				return syncInfo
			}

			syncInfo = ctrl.cleanupSnapshot(vmClone, syncInfo)
			if syncInfo.toReenqueue() {
				return syncInfo
			}
		}
```
If we want to drop the reset of snapshot and restore name from the status, I should also add a new field, i.e as `pvcBound`.

Reply:
Or handle it with a condition

Reply:
I am not sure if the continuous verification would be a problem. In theory, once you move the `clone` to `Succeeded` there should not be many more `syncs`. But just to note, this is out of scope for the PR.

Reply:
Will try to think about it for a follow-up, thanks!